modules/examiner/php/editexaminer.class.inc

```diff
@@ -265,10 +265,7 @@ class EditExaminer extends \NDB_Form
             }
         }
 
-        $baseurl = \NDB_Factory::singleton()->settings()->getBaseURL();
-        $url     = "{$baseurl}/examiner/editExaminer/?identifier="
-               . $examinerID;
-        header("Location: " . $url, true, 303);
+        header("Refresh:0");
```

Comment:
Why is this changed? Neither usage of `header` is really good practice, but responding to a post request with a 303/location is still good practice..

Reply:
Hi @driusan, I give here the same solution given in PR #5738 which seem to be merged into `aces/master` already.  The code in red (removed) was not allowing the save button to refresh once clicked.
![image](https://user-images.githubusercontent.com/37309344/83766150-09c64180-a64a-11ea-9d60-10f055b45e65.png)